org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -45,9 +47,23 @@ public String handleRequest(Map<String, String> lambdaInput, Context context) {
     final AmazonSQS sqsClient = AmazonSQSClientBuilder.defaultClient();
     final AWSInputAdapter inputAdapter = new AWSInputAdapter(sqsClient, queueUrl);
 
+    // Create file manager
+    final UserProjectFileManager userProjectFileManager =
+        new UserProjectFileManager(projectUrl, fileNames);
+
     // Create and invoke the code execution environment
-    JavaBuilder javaBuilder = new JavaBuilder(inputAdapter, outputAdapter);
-    javaBuilder.runUserCode();
+    try {
+      CodeBuilder codeBuilder =
+          new CodeBuilder(inputAdapter, outputAdapter, userProjectFileManager);
+      codeBuilder.runUserCode();
```

Comment:
Do we need to call cleanUp somewhere in this function?  (Or maybe that's future work?)

Reply:
Great call out. I spent a lot of time scratching my head yesterday trying to figure out why cleanup wasn't working when I tried it in AWS :)